tools/7754F0_old.c

```diff
@@ -1,6 +1,6 @@
 #include "actor.h"
-#include "data_symbols.h"
-#include "function_symbols.h"
+
```

Comment:
Was this a mistake? You can ignore files locally by adding them to the exclude file